### PR TITLE
feat: add RPC get_block_median_time

### PR DESCRIPTION
**Pull request by keroro520**
resolve https://github.com/nervosnetwork/ckb/issues/2518

### CODE REVIEW

File: rpc/src/module/chain.rs [hunk 1]
```diff
@@ -1193,6 +1194,46 @@ pub trait ChainRpc {
     /// ```
     #[rpc(name = "get_consensus")]
     fn get_consensus(&self) -> Result<Consensus>;
+
+    /// Returns the past median time by block hash.
+    ///
+    /// ## Params
+    ///
+    /// * `block_hash` - the highest block hash
```
Comment (chuijiaolianying) [inline rpc/src/module/chain.rs:1202]: I think `the highest block hash` may create confusion about this param should be the tip block hash or block hash of any height in the chain

Comment (keroro520) [reply rpc/src/module/chain.rs:1202]: Thanks. I agree with that. What about the below, or any suggestions?

> "A median time is calculated for a consecutive block sequence. `block_hash` indicates the highest block of the sequence."

Comment (chuijiaolianying) [reply rpc/src/module/chain.rs:1202]: LGTM

Comment (keroro520) [reply rpc/src/module/chain.rs]: Updated document.

**Review by chuijiaolianying — APPROVED** [review]

**Review by chuijiaolianying — APPROVED** [review]

**Review by quake — APPROVED** [review]

**Comment by yangby-cryptape** [comment]
bors r=chuijiaolianying,quake

**Pull request merged**